draft/draft-ietf-idr-bgp-model.xml

```diff
@@ -655,10 +658,10 @@ reference: RFC XXXX
           context.</li>
 
           <li>ietf-bgp-rib - grouping for representing BGP RIBs.</li>
-	  <li>ietf-bgp-rib-attributes - common data definitions for BGP
-	  attributes used in BGP RIB tables.</li>
-	  <li>ietf-bgp-rib-tables - structural data definitions for BGP
-	  routing tables.</li>
+          <li>ietf-bgp-rib-attributes - common data definitions for BGP
```

Comment:
I do not understand these changes. Are these formatting changes?

Reply:
Errant tabs to spaces changes throughout the file.  We really need something in the build that catches it.